Microsoft.Dynamics365.UIAutomation.Api/Browser.cs

```diff
@@ -175,6 +175,12 @@ public void GoToXrmUri(Uri xrmUri)
         public Office365Navigation Office365Navigation => this.GetPage<Office365Navigation>();
 
         #endregion Office365
+
+        #region Pending Email Reminder
+
```

Comment:
I believe we have a Dialogs page that has a CloseWarningDialog method that will handle the Pending Email warning.  I would like to be consistent with how we handle Dialogs and not introduce complexity of individual pages for each.   Can we test the existing CloseWarningDialog or perhaps move the methods to the Dialogs page?

Reply:
I thought about it but decided to put this methods to separate page for some reason I can't remember. I'll try CloseWarningDialog and come back with comments.

Reply:
Yes you are right. CloseWarningDialog works as expected so this PR is not needed